ULiege-Droit.csl

```diff
@@ -0,0 +1,520 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" default-locale="fr-FR" version="1.0" demote-non-dropping-particle="never">
+  <info>
+    <title>ULiège - Droit (French)</title>
+    <title-short>ULiège-dt</title-short>
+    <id>http://www.zotero.org/styles/ULiege-Droit</id>
+    <link href="http://www.zotero.org/styles/universite-libre-de-bruxelles-histoire" rel="template"/>
```

Comment:
you need a self link here that matches the id and also has the same filename then.
See how it's done in your template style: https://github.com/citation-style-language/styles/blob/master/universite-de-liege-histoire.csl
It should be the full name. Universite the Liege - Droit (French) and then <id>http://www.zotero.org/styles/universite-de-liege-droit</id> and the same for the self link